examples/utils/migration-tools/package.json

```diff
@@ -0,0 +1,102 @@
+{
+	"name": "@fluid-example/migration-tools",
+	"version": "2.3.0",
+	"private": true,
+	"description": "Tools for migrating data",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "examples/utils/migration-tools"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"sideEffects": false,
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.js"
+			}
+		}
+	},
+	"main": "lib/index.js",
+	"types": "lib/index.d.ts",
+	"scripts": {
+		"build": "fluid-build . --task build",
+		"build:compile": "fluid-build . --task compile",
+		"build:docs": "api-extractor run --local",
+		"build:esnext": "tsc --project ./tsconfig.json",
+		"check:are-the-types-wrong": "attw --pack .",
+		"check:biome": "biome check .",
+		"check:exports": "api-extractor run --config ./api-extractor-lint.json",
+		"check:format": "npm run check:biome",
+		"check:prettier": "prettier --check . --cache --ignore-path ../../../.prettierignore",
+		"ci:build:docs": "api-extractor run",
```

Comment:
Remove "build:docs" and "ci:build:docs" to avoid API extractor reports. `api-extractor.json` could also be removed.
I think the policy will be okay with you leaving the API linting via "check:exports": "api-extractor run --config ./api-extractor-lint.json" using api-extractor-lint.json.
All of the APIs are tagged `@internal` which doesn't seem to make sense. Add notes if this is intentional and explain the pattern.
- If nothing will ever be publicly used (package could be in the `@fluid-internal` scope eventually) then remove all API related things - linting too.
- If some might be public facing, then use a tag other than `@internal` to distinguish. The linting step should keep tagging honest (even without the report).